src/query/atomWithQuery.ts

```diff
@@ -156,7 +170,6 @@ export function atomWithQuery<
             }
           }, 1000)
           resolve(result)
-          resolve = null
```

Comment:
This seems bad. (Interesting all tests pass... 🤔

Reply:
Removing it will make the tests fail!

Reply:
in versionedWrite

Reply:
hmm, there might be something wrong in other places..

Reply:
We don't have a test to catch if we remove `resolve = null`.
It's a test to update result from queryClient, I suppose.